src/Components/AddTask/AddTask.tsx

```diff
@@ -1,68 +1,126 @@
+import { ToastContext } from '../../Context/ToastContext'
 
 export default function AddProject() {
- const navigate= useNavigate()
-const {register,handleSubmit,formState:{errors}}=useForm()
-let{baseUrl,reqHeaders}:any=useContext(AuthContext)
-const AddProject =(data)=>{
-axios.post(`${baseUrl}/Task`,data,{headers:reqHeaders}).then((response)=>{
-  
-  navigate(-1)
-  
-}).catch((error)=>{
-  console.log(error);
-  
-})
+  const navigate = useNavigate()
+  const { register, handleSubmit, formState: { errors } } = useForm()
+  let { baseUrl, reqHeaders }: any = useContext(AuthContext)
+  let { getToastValue } = useContext(ToastContext)
+  const [users, setUsers] = useState([])
+  const [allProjects, setAllProjects] = useState([])
+ 
 
-}
+
+  {/* Add  Task*/ }
+  const AddTask = (data) => {
+    axios.post(`${baseUrl}/Task`, data, { headers: reqHeaders }).then((response) => {
+   
+
+      navigate(-1)
+
+    }).catch((error) => {
+
+      getToastValue("error", error?.response?.data?.message)
+
+    })
+
+  }
+
+  {/* Get  Users*/ }
+  const Users = () => {
+    axios.get(`${baseUrl}/Users/Manager?pageSize=10&pageNumber=1`, { headers: reqHeaders }).then((response) => {
+      setUsers(response?.data?.data)
+
+    }).catch((error) => {
+      getToastValue('error', error?.response?.data?.message)
+
+    })
+  }
+
+  {/* Get  All Projects*/ }
+  const AllProjects = () => {
+    axios.get(`${baseUrl}/Project/?pageSize=20&pageNumber=1`, { headers: reqHeaders }).then((response) => {
+      setAllProjects(response.data.data);
+
+
+
+    }).catch((error) => {
+      getToastValue('error', error?.response?.data?.message);
+    })
+  }
+
+
+
+  useEffect(() => {
+    Users();
+    AllProjects();
+  }, [])
   return (
     <>
-    <form onSubmit={handleSubmit(AddProject)} >
-    <div className="container bg-white addHead pb-4">
-    <div className="col-md-12">
-      <div>
-        <h5 className="p-3 m-auto" onClick={()=>navigate(-1)}>
-        <i className="fa-solid fa-chevron-left pe-2"> </i>
-        view All Tasks
-        </h5>
-      <h3> New Task</h3>
-      </div>
-    </div>
-  </div>
-  <div className="container">
-<div className="col-md-8  m-auto bg-white mt-4  p-5 rounded-3">
-  <div className="inputs m-auto ">
-    <label className='d-block fw-bolder ' >Title</label>
-    <input type="text" placeholder='Name' className='form-control  border-2 rounded-2 ' 
-    {...register('title',{required:true})}
-    />
-    </div>
-    <div className='w-100 p-3 m-auto text'>
-
-    
-    <label className='d-block fw-bolder' >Description</label>
-  
-   <textarea className='form-control  border-2 rounded-2' cols="20" rows="4"   {...register("description",{required:true})} >Description
- 
+      <form onSubmit={handleSubmit(AddTask)} >
+        <div className="container bg-white addHead pb-4">
+          <div className="col-md-12">
+            <div>
+              <h5 className="p-3 m-auto" onClick={() => navigate(-1)}>
```

Comment:
This should be replaced by a Link Component from the React-Router package, as the main purpose of a Link is to navigate between pages.
and the styles of it need to change to make it feel like a button to indicate that it is clickable.